### DISCUSSION TITLE
Using Pip and Python from Docker

### DISCUSSION THREAD

**Discussion by Surgical17** [coder/code-server#6872] [General]
Hi All

In short, I am having trouble setting up the Code server. I managed to add a volume to point to where my python3.12 is installed on my host machine, but I am having trouble using it and pip.

The interpreter was added and picked up but Bash can't find the python3 command nor can I use or even install pip3.

All I want to do is use Code Server to code/manage and run my Python scripts on my host or if I really have to from within the docker container.

Any advice?

**Comment by code-asher** [comment]
How are you mounting the volume?

It is probably better to install Python and pip in the Docker container instead of mounting Python into the container, though.  The host Python might not be compatible with the container.
